src/__tests__/test-process-tpl.js

```diff
@@ -13,6 +13,7 @@ test('test process-tpl', () => {
 		'<link rel="preload" href="//gw.alipayobjects.com/as/g/antcloud-fe/antd-cloud-nav/0.2.22/antd-cloud-nav.min.js">\n' +
 		'<link rel="prefetch" href="/a3-ie6-polyfill.js">\n' +
 		'<link rel="stylesheet" href="/umi.css">\n' +
+		'<link rel="preload" as="font" href="/static/fonts/iconfont.woff" type="font/woff" crossorigin="anonymous">\n' +
```

Comment:
need to check font not be replaced after processed

Reply:
yes, I checked it by ‘console.log(template)‘

Reply:
Should be checked with an assert code rather than a manual work